webaverse/app
# First person model clipping / FPS neck model

Now we are using Neck model and point between eyes for camera position.
Neck is rotated forward with mouse movement.

Related issue:
https://github.com/webaverse/app/issues/1609

Steps for testing:
-Zoom in to first person view
-Move mouse down to see avatars body
-Camera should not look inside avatar
